packages/lesswrong/components/collections/CollectionsCard.jsx

```diff
@@ -17,31 +17,39 @@ const styles = theme => ({
   }
 })
 
-const CollectionsCard = ({collection, big = false, url, classes}) => {
-  const cardContentStyle = {borderTopColor: collection.color}
+class CollectionsCard extends PureComponent {
+  handleClick = (event) => {
+    const { url, router } = this.props
+    if (!event.target.closest('a')) { // Checks that no parent of the clicked element is a tag
```

Comment:
`Element.closest()` is not supported in IE. I suggest instead writing a loop to manually search up the tree by looking at `node.parentElement`.

See https://developer.mozilla.org/en-US/docs/Web/API/Element/closest for "closest" browser support